README.md

```diff
@@ -24,3 +24,59 @@ See `const thumber` in `scthumbd.js` for now...
 ```
 npm run scthumbd
 ```
+
+
+## Envrionment variables
+
+`PORT` - listening port, defaults to 4001.
+
+`WORKERS` - number of worker processes to spawn, defaults to number of cpus detected.
+
+
+## Docker
+
+This assumes Docker Compose V2 is used - replace `docker compose` with `docker-compose` if using Docker Compose V1.
+
+Running for development:
+```
+docker compose up
+```
+will mount the local directory to the container; code changes will be reflected when the contianer is restarted.
+The container will accept requests on port 4001. To use a different port, provide a `HTTP_PORT` env, e.g.:
+
+```
+HTTP_PORT=8001 docker compose up
+```
+
+To override settings inside the container such as the number of worker or listening port, it is recommended to create another docker-compose file with the extra settings and use that file when running `docker compose`, e.g.:
+
+```
+docker compose -f docker-compose.yml -f docker-compose.dev.yml up
+```
+
+
+To just build an image
+```
+docker build -t <repository_name>:<tag> .
+```
+
+### Networking Notes
+
+This assumes the default Docker bridge networking is used; different networking modes and container runtimes may require different configuration.
+
+scthumber needs to be able to download the image source for thumbnailing.
+If using a custom internal domain that doesn't resolve to a public IP address for the image origin, you will need to add a mapping to the container to resolve.
+
+Get host IP address:
+```
+docker run --rm scthumber /bin/sh -c 'echo "$(getent hosts host.docker.internal)"' | awk '{ print $1 }'
```

Comment:
doesn't work here (arch linux)  
![Screenshot_20211029_203222](https://user-images.githubusercontent.com/4985725/139484927-1e6c8ab7-17a9-43b6-8aa9-c5f7ce5cca6e.png)

Reply:
turns out this is a Docker Desktop specific feature 🤔